core/src/main/java/nl/nn/adapterframework/http/rest/ApiListenerServlet.java

```diff
@@ -529,14 +530,7 @@ else if(segment.startsWith("{") && segment.endsWith("}")) {
 				/*
 				 * Calculate an eTag over the processed result and store in cache
 				 */
-				Boolean updateEtag = messageContext.getBoolean(UPDATE_ETAG_CONTEXT_KEY);
-				if (updateEtag==null) {
-					updateEtag=listener.getUpdateEtag();
-				}
-				if (updateEtag==null) {
-					updateEtag=result==null || result.isRepeatable();
-				}
-				if(updateEtag) {
+				if (Boolean.TRUE.equals(messageContext.getBoolean(UPDATE_ETAG_CONTEXT_KEY))) {
```

Comment:
Wordt verwacht dat dit in de messageContext veranderd kan worden, tussen regel 414 en dit punt? Door de adapter/pipeline, of door een query-parameter genaamd `updateEtag`?

Reply:
In de pipeline kan het 'dynamisch' aangepast worden.